packages/web3-providers-http/test/integration/request.test.ts

```diff
@@ -0,0 +1,34 @@
+import {
+	EthExecutionAPI,
+	Web3APIPayload,
+	JsonRpcResponse,
+	JsonRpcResponseWithResult,
+} from 'web3-common';
+import { toWei } from 'web3-utils';
+import HttpProvider from '../../src/index';
+import { accounts, clientUrl } from '../fixtures/config';
+
+describe('HttpProvider - implemented methods', () => {
+	const account = accounts[0];
+	const jsonRpcPayload = {
+		jsonrpc: '2.0',
+		id: 42,
+		method: 'eth_getBalance',
+		params: [account.address, 'latest'],
+	} as Web3APIPayload<EthExecutionAPI, 'eth_getBalance'>;
```

Comment:
Always initialize test fixtures in `beforeEach`, to avoid any side effect of value mutation.

Reply:
fixed